src/avram/queryable.cr

```diff
@@ -243,6 +243,22 @@ module Avram::Queryable(T)
     end
   end
 
+  def group_count
+    database.query_all(
+      query.select_direct(query.groups + ["COUNT(*)"]).statement,
+      args: query.args,
+      queryable: schema_class.name,
+    ) do |rs|
+      k = query.groups.map do
+        rs.read PG::PGValue | Int64 # TODO: Why isn't Int64 part of PGValue?
```

Comment:
@jwoertink @robcole and I were talking about this, not sure why it's not a part of PG::PGValue. Maybe @will knows?

Reply:
I think it may just be an oversight. I'd open up a PR adding that, plus the other supported Ints.